developer-docs-site/docs/tutorials/testnet-devnet-incentivized-testnet.md

```diff
@@ -0,0 +1,25 @@
+---
+title: "Testnet, Devnet and Incentivized Testnet"
+slug: "testnet-devnet-and-incentivized-testnet"
+---
+
+# Testnet, Devnet and Incentivized Testnet
```

Comment:
nit: can we change "Testnet" to "Local Testnet" throughout the document? I think it'll make things much cleaner.